Components/cards.css

```diff
@@ -1,9 +1,4 @@
  /* ----- Horizontal Cards----- */
- .card-display{
-    padding: 1rem;
-    border: 1px solid var(--color-gray-200);
-    width: 60%;
-  }
   .horizontal-card-content > *{
     margin:0px;
```

Comment:
I think this is not needed

Reply:
removing this is taking the card contents outside card.